flocker/provision/_install.py

```diff
@@ -743,7 +754,12 @@ def _remove_private_key(content):
     if trim_end <= trim_start:
         # strangely short key, keep all content
         return content
-    return content[:trim_start] + '...REMOVED...' + content[trim_end:]
+    end_of_key = end+len(suffix)
+    trimmed_key = (content[:trim_start] +
+                   '...REMOVED...' +
+                   content[trim_end:end_of_key])
+    # keep looking for more keys
+    return trimmed_key + _remove_private_keys(content[end_of_key:])
```

Comment:
Probably uncecessary but updated this to remove multiple keys from the output (used to stop after a single key)